models/health.go

```diff
@@ -74,53 +74,33 @@ type WorkloadStatus struct {
 
 // RequestHealth holds several stats about recent request errors
 type RequestHealth struct {
-	inboundErrorRate    float64
-	outboundErrorRate   float64
-	inboundRequestRate  float64
-	outboundRequestRate float64
-
-	ErrorRatio         float64 `json:"errorRatio"`
-	InboundErrorRatio  float64 `json:"inboundErrorRatio"`
-	OutboundErrorRatio float64 `json:"outboundErrorRatio"`
+	Inbound  map[string]map[string]float64 `json:"inbound"`
```

Comment:
When introducing nested maps, it'd be good to describe in a comment the potential values for keys and other relevant info in the structure (what values should or shouldn't be there).